jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ProjectProperties.java

```diff
@@ -54,32 +78,29 @@ String getMainClass(@Nullable String mainClass) {
         gradleBuildLogger.debug(
```

Comment:
I was talking to @patflynn and Patrick mentioned that it might be better to log something like `Searching for main class... Add a 'mainClass' configuration to 'jib' to improve build speed.` at the `info` level to help users in case they have a ton of classes to search.